metricbeat/docs/gettingstarted.asciidoc

```diff
@@ -267,13 +280,17 @@ docker run {dockerimage}
 [source,shell]
 ----------------------------------------------------------------------
 sudo chown root metricbeat.yml <1>
-sudo ./metricbeat -e -c metricbeat.yml -d "publish"
+sudo chown root modules.d/system.yml <1>
+sudo ./metricbeat run -e -c metricbeat.yml -d "publish"
 ----------------------------------------------------------------------
 <1> You'll be running Metricbeat as root, so you need to change ownership
-of the configuration file (see
+of the configuration file and any configurations enabled in the `modules.d`
+directory (see
 {libbeat}/config-file-permissions.html[Config File Ownership and Permissions]
 in the _Beats Platform Reference_).
 
+//REVIEWERS: It seems really inconvenient to users to expect them to change ownership on all the files in `modules.d`. This is something I had to do, but not 100% sure if there is a better way. Please advise.
+
```

Comment:
I think using the `strict.perms` is fine here. I'd explain why it's used and link over to docs for the option.

I made this [comment](https://github.com/elastic/beats/issues/4567#issuecomment-311525217) previously, and I still think we should consider it.

> Another option would be to make `-strict.perms=false` the new default. Then modify the scripts that start Beats as services to enable the checks. I liken this to how ES disables bootstrap checks for non-production mode.